for-wp-developers/internal.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Internal Reuse
+---
+
+If you're writing code to work with the REST API, but still want things handled in PHP, you might notice that you need to duplicate code to handle differences between the API and internal functions. The REST API provides an internal, PHP-based API that you can use to do everything you do through the external, HTTP-based API.
+
+The key function for performing internal requests is [`rest_do_request()`](https://developer.wordpress.org/reference/functions/rest_do_request/). This function takes a [`WP_REST_Request` object](https://developer.wordpress.org/reference/classes/wp_rest_request/) and returns a [`WP_REST_Response` object](https://developer.wordpress.org/reference/classes/wp_rest_response/). You can use these to 
+
+For example, you may have code that renders a post into HTML, and want to use this on the server as well. Let's say your frontend code looks something like this:
+
+```js
+$.ajax( '/wp/v2/posts/42', {
```

Comment:
I'm assuming most WordPress developers aren't familiar with `fetch` and also aren't going to be any time soon.